specs/commandline/sign.md

```diff
@@ -27,58 +41,78 @@ Flags:
 
 ## Usage
 
-### Sign a container image
+### Sign an OCI artifact
 
 ```shell
-# Add a key which uses a local private key and certificate, and make it a default signing key
-notation key add --default --name <key_name> <key_path> <cert_path>
-
-# Or change the default signing key to an existing signing key
-notation key update --default <key_name>
+# Prerequisites: 
+# - A signing plugin is installed. See plugin documentation (https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md) for more details.
```

Comment:
I have two concerns:
1. A signing plugin is not a hard prerequisite for all cases, only signing with the key stored in the KMS system.
2. This link is not a good reference for installing a plugin.

Reply:
- We don't support local signing scenario in rc.1
- We can replace it with an installation guide, but we don't have it yet. The link here is for user to learn what is a plugin